xenia-build

```diff
@@ -389,7 +389,7 @@ def get_build_bin_path(args):
         platform = 'windows'
     else:
         platform = 'linux'
-    return os.path.join(self_path, 'build', 'bin', platform, args['config'])
+    return os.path.join(self_path, 'build', 'bin', platform.capitalize(), args['config'].capitalize())
```

Comment:
@gibbed Does this matter?

Reply:
I think I merged an extra commit. How do I undo it?

Reply:
In this case the change is simple enough that you can manually undo, add, and commit the change. Then squash the commit. It'll appear as never have been changed.